xml/depl_cisco_opflex.xml

```diff
@@ -0,0 +1,686 @@
+   
+   <step>
+    <para>
+       Select the Ethernet device that is connected to the ACI fabric  for <literal>Real Interface for VLAN</literal>.
+    </para>
+   </step> 
+   
+    <step>
+    <para>
+       Enter your VLAN ID, which in the example network is 4093.
+    </para>    
+   </step> 
+    
+   <step>
+    <para>
+      Enable <literal>Dynamic Address - DHCP</literal>.
+    </para>
+   </step> 
+   
+    <step>
+    <para>
+      Change to the General tab and set the MTU to 1600.
+    </para>
+   </step> 
+
+    <step>
+    <para>
+      Press F10 to save changes for new VLAN device, and press F10 again to exit Network Settings.
+    </para>
+   </step>    
+</procedure>
+
+
+<para>
+   Now that you have made these changes, re-enter Network Settings and configure routing. 
+</para>
+
+<procedure>
+   <step>
+    <para>
+     Go back into Network Settings (Click Continue when you see the Chef client warning).
+    </para>
+   </step> 
+   
+   <step>
+    <para>
+        Change to the Routing tab and add these entries to the Routing Table:
+        
+        <command>Destination     224.0.0.0
+        Genmask         240.0.0.0
+        Gateway         0.0.0.0
+        Device          vlan.4093 </command>(Select the VLAN device created in the previous steps)
+    </para>
+   </step> 
+   
+    <step>
+    <para>
+     Press F10 to save and exit Network Settings, then press F9 to exit Yast.
+    </para>
+   </step> 
+ </procedure>
+</sect1>
+
+<sect1 xml:id="sec.connectivity">
+  <title>Confirm Correct Connectivity</title>
+
+  <para>
+      Edit the <filename>/etc/dhclient.conf</filename> file on all compute nodes to verify that <literal>send dhcp-client-identifier</literal> has the MAC address of the NIC on the ACI fabric. (Run <command>ip addr show</command> to find the MAC address.) It should look like this, uncommented, with <literal>1:</literal> prepended to your MAC address, followed by a semi-colon, like this example: 
+   </para>
+   
+    <para> 
+        <literal>  
+     send dhcp-client-identifier 1:0:a0:24:ab:fb:9c;
+     </literal>
+  </para>
+  
+  <para>
+     Now install and run <command>lldpd</command>, the Link Layer Discovery Protocol daemon. You will use this to verify correct connectivity between your compute nodes.
+ </para>
+ 
+ <screen><prompt>root #  </prompt><command>zypper in lldpd</command> 
+<prompt>root #  </prompt><command>systemctl start lldpd.service</command> 
+<prompt>root #  </prompt><command>lldpctl</command> </screen>
+
+<para>
+    Give it a few minutes to scan its LLDP neighbors. When the scan is completed make a note of the <literal>PortDescr:</literal> line for the Ethernet interface(s) on the ACI fabric. It should look like this example, with the nodes and port switch detailed:
+</para>   
+  
+<para>
+    <literal>
+    PortDescr: topology/pod-1/paths-102/pathep-[eth1/34]
+    </literal>
+</para> 
+
+<warning>
+    <title>Do not proceed if you do not see proper output from lldpctl</title>
```

Comment:
we use capitalized <titles> -> <title>Do Not Proceed If You Do Not See Proper Output from <command>lldpctl</command></title>